website/pages/style.css

```diff
@@ -101,3 +101,7 @@
 .g-content {
   --brand-text: var(--vagrant) !important;
 }
+.g-docs-sidenav {
+  --brand: var(--vagrant) !important;
+  --brand-text: var(--vagrant) !important;
+}
```

Comment:
This is very similar to the override above - we're in a bit of a transient branding stage, and with Vault specifically the `--brand` colors actually cause quite a few issues. There is a "debt" comment just above here that I'm rolling this change into.

In this specific case, the `--brand` value for vault is gray by default, so "active" sidebar items, which should appear highlighted, are really hard to pick out.

This change is kind of unrelated to the "unlinked pages" issue; I've added it in because it felt important to patch for UX purposes.